exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/BaseParquetMetadataProvider.java

```diff
@@ -485,6 +486,7 @@ private static SegmentMetadata combineSegmentMetadata(Collection<SegmentMetadata
       if (entries.isEmpty() || !collectMetadata) {
         return Collections.emptyMap();
       }
+      @SuppressWarnings("unused")
```

Comment:
`addRowGroups` is used in the code below, so perhaps it is your IDE false-positive warning, IDEA doesn't highlight it as unused.

Reply:
Eclipse claims this is dead code, but I can't figure out why. So, I'll just remove the annotation and leave well-enough alone.